myia/anf_ir.py

```diff
@@ -292,3 +362,9 @@ class Constant(ANFNode):
     def __init__(self, value: Any) -> None:
         """Construct a literal."""
         super().__init__([], value, None)
+
+    def __str__(self) -> str:
+        return str(self.value)
+
+    def __repr__(self) -> str:
+        return repr(self.value)
```

Comment:
`<Parameter(value={self.value}) at ...>` or something would be better, else you can't distinguish between `repr(p)` and `repr(p.value)` when debugging.

Reply:
Good point.